client/src/components/MileageModal.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <v-dialog v-model="value" :fullscreen="isFullscreen" width="500px">
+    <v-card height="600px" class="bg-backgroundGrey">
+      <div style="height: 10px">
+        <v-img src="/images/Footer-min.jpeg" width="100%" cover />
+      </div>
+      <v-spacer />
+      <v-form v-model="form">
+        <v-container class="px-8">
+          <v-row align="center">
+            <v-col>
+              <v-card-title class="mx-auto text-h5 justify-center">Add Mileage</v-card-title>
```

Comment:
It would be cool to add a subtitle underneath the title in a lighter grey colour with something like "Let us know how far you travelled today!" You might have a better message to add instead, thats just from the top of head and I just woke up XD